opts/hosts_test.go

```diff
@@ -8,7 +8,7 @@ import (
 func TestParseHost(t *testing.T) {
 	invalid := map[string]string{
 		"anything":              "Invalid bind address format: anything",
-		"something with spaces": "Invalid bind address format: something with spaces",
+		"something with spaces": "parse tcp://something with spaces: invalid character \" \" in host name",
```

Comment:
nope, they really change that message :pensive: Before:

```
FAIL: TestParseHost (0.00s)
	hosts_test.go:42: Expected an error for something with spaces with [Invalid bind address format: something with spaces], got [parse tcp://something with spaces: invalid character " " in host name]
```

You can see that janky is not failing in that test.